modules/ingester/instance_search.go

```diff
@@ -399,7 +367,9 @@ func (i *instance) SearchTagValuesV2(ctx context.Context, req *tempopb.SearchTag
 	query := extractMatchers(req.Query)
```

Comment:
Having trouble with the query `{ resource.service.name="shop-backend" && span.http.method="PUT" && kind=server && span.http.url= }`.   It fails with a user-facing UI error `unexpected error searching complete block (ac7f17fd-033f-4ff3-816a-9aeaf90edc8c): parse error at line 1, col 73: syntax error: unexpected IDENTIFIER`

extractMatchers returns `{resource.service.name="shop-backend" && span.http.method="PUT" && kind=s}` so I think the issue is with the unquoted `=server`.

Reply:
Fixed the regex to support those queries, nice catch.